.pre-commit-config.yaml

```
@@ -16,3 +16,11 @@ repos:
     rev: 6.0.0
     hooks:
       - id: pydocstyle
+  # Using system installation of pylint to support checking python module imports
+  - repo: local
+    hooks:
+      - id: pylint
+        name: pylint
+        entry: pylint
+        language: system
+        types: [python]
```

```suggestion
        types: [python]
        args:
            [
              "-rn", # Only display messages
              "-sn", # Don't display the score
              "--rcfile=pylintrc", # Link to pylint config file
            ]
```
I took these lines out of the pylint docs; I'd like pylint to point to our pylintrc

---

It automatically finds the `.pylintrc` file (the default location). The report should be disabled by default, but happy to explicitly disable the report / score, added a commit to address this.

---

cool, I didn't know that -- thanks for letting me know!